pages/docs/tutorials/kotlin-android.md

```diff
@@ -7,51 +7,56 @@ authors:
 showAuthorInfo: false
 ---
 
-It’s extremely easy to start using Kotlin for Android development.
-In this tutorial we’ll follow the warming up process with Android Studio. If you're using Intellij IDEA with Android, the process is almost the same.
+使用 Kotlin 进行 Android 开发非常简单。
 
```

Comment:
Wired, there is no such blank line in Markdown, but it displays on Github, I need to find the reasons.